pkg/statdb/sdbclient/client.go

```diff
@@ -26,13 +26,16 @@ type StatDB struct {
 // Client services offerred for the interface
 type Client interface {
 	Create(ctx context.Context, nodeID []byte) error
+	CreateWithStats(ctx context.Context, nodeID []byte, auditCount, auditSuccessCount,
+		uptimeCount, uptimeSuccessCount int64) error
```

Comment:
I could probably actually receive a pb.Node for everything, since that includes a nodeID and nodestats. I guess I was trying to separate the statdb protos from the client interface, but with enough args that becomes inconvenient